Adyen.docc/DropIn.md

```diff
@@ -31,63 +56,51 @@ let applePayment = try ApplePayPayment(countryCode: "US",
                                        currencyCode: "USD",
                                        summaryItems: summaryItems)
 
-let configuration = DropInComponent.Configuration(apiContext: apiContext)
+let dropInConfiguration = DropInComponent.Configuration(context: context)
 configuration.applePay = .init(payment: applePayment,
                                merchantIdentifier: "merchant.com.adyen.MY_MERCHANT_ID")
 ```
 
-Also for voucher payment methods like Doku variants, in order for the `DokuComponent` to enable the shopper to save the voucher, access to the shopper photos is requested, so a suitable text need to be added to key  `NSPhotoLibraryAddUsageDescription` in the application Info.plist.
-
-After serializing the payment methods and creating the configuration, the Drop-in is ready to be initialized. Assign a `delegate` and use the `viewController` property to present the Drop-in on the screen:
-
-```swift
-let dropInComponent = DropInComponent(paymentMethods: paymentMethods, configuration: configuration)
-dropInComponent.delegate = self
-present(dropInComponent.viewController, animated: true)
-```
-#### Implementing DropInComponentDelegate
+Also for voucher payment methods like Doku variants, in order for the `DokuComponent` to enable the shopper to save the voucher, access to the shopper photos is requested, so a suitable text needs to be added to the `NSPhotoLibraryAddUsageDescription` key in the application `Info.plist`.
 
-To handle the results of the Drop-in, the following methods of `DropInComponentDelegate` should be implemented:
+### Presenting the Drop-in
 
----
+Initialize the `DropInComponent` class and set the `AdyenSession` instance as the `delegate` and `partialPaymentDelegate` (if needed) of the `DropInComponent` instance.
 
 ```swift
-func didSubmit(_ data: PaymentComponentData, for paymentMethod: PaymentMethod, from component: DropInComponent)
-```
-
-This method is invoked when the customer has selected a payment method and entered its payment details. The payment details can be read from `data.paymentMethod` and can be submitted as-is to `/payments`.
-
----
+let dropInComponent = DropInComponent(paymentMethods: session.sessionContext.paymentMethods,
```

Comment:
Just a thought: 
should we have a separate doc for sessions?

Reply:
Session is the default way moving forward. And docs will have docs for both.